modules/scripts/config.py

```diff
@@ -101,6 +102,7 @@ def _get_env_dict(conf):
     build = conf['build']
     publish = conf['publish']
     extras = conf['extras']
+    social = conf['social']
```

Comment:
Initially I submitted PR like that only but ujwal wanted this feature. if he okay then I'll remove. @imujjwal96 are you okay with this?